SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/ManagedWorkerCoordinator.cs

```diff
@@ -136,21 +156,50 @@ public override void Run()
             }
 
             Array.Sort(startDelaysMillis);
-            for (int i = 0; i < NumSimulatedPlayersToStart; i++)
+
+            if (LifetimeComponent == null)
```

Comment:
I'd unify both of these under the `LifetimeComponent`.

Reply:
Good, I did this because I wanted to try not to modify the logic of the original part. Now it seems that the logic of the old and new parts are indeed very similar. I will unify these in `LifetimeComponent`.